apps/web/pages/v2/settings/my-account/appearance.tsx

```diff
@@ -46,129 +46,133 @@ const AppearanceView = (props: inferSSRProps<typeof getServerSideProps>) => {
   const formMethods = useForm();
 
   return (
-    <Form
-      form={formMethods}
-      handleSubmit={(values) => {
-        mutation.mutate({
-          ...values,
-          theme: values.theme.value,
-        });
-      }}>
-      <Controller
-        name="theme"
-        control={formMethods.control}
-        defaultValue={user.theme}
-        render={({ field: { value } }) => (
-          <>
-            <div className="flex items-center">
-              <div>
-                <p className="font-semibold">{t("follow_system_preferences")}</p>
-                <p className="text-gray-600">
-                  <Trans i18nKey="system_preference_description">
-                    Automatically adjust theme based on invitee system preferences. Note: This only applies to
-                    the booking pages.
-                  </Trans>
-                </p>
-              </div>
-              <Switch
-                onCheckedChange={(checked) => formMethods.setValue("theme", checked ? null : themeOptions[0])}
-                checked={!value}
-              />
-            </div>
-            <div>
-              <Select
-                options={themeOptions}
-                onChange={(event) => {
-                  if (event) formMethods.setValue("theme", { ...event });
-                }}
-                isDisabled={!value}
-                defaultValue={value || themeOptions[0]}
-                value={value || themeOptions[0]}
-              />
-            </div>
-          </>
-        )}
-      />
-
-      <hr className="border-1 my-8 border-neutral-200" />
-      <div className="mb-6 flex items-center">
-        <div>
-          <p className="font-semibold">{t("custom_brand_colors")}</p>
-          <p className="text-gray-600">{t("customize_your_brand_colors")}</p>
-        </div>
-      </div>
-
-      <div className="flex justify-between">
-        <Controller
-          name="brandColor"
-          control={formMethods.control}
-          defaultValue={user.brandColor}
-          render={({ field: { value } }) => (
-            <div>
-              <p className="block text-sm font-medium text-gray-900">{t("light_brand_color")}</p>
-              <ColorPicker
-                defaultValue={user.brandColor}
-                onChange={(value) => formMethods.setValue("brandColor", value)}
-              />
-            </div>
-          )}
-        />
+    <SettingsShell heading={t("appearance")} subtitle={t("appearance_subtitle")}>
```

Comment:
Let’s stick with the getLayout approach, if that layout doesn’t cover the current needs let’s modify it or make another one.